.github/workflows/CI.yml

```diff
@@ -69,12 +56,14 @@ jobs:
           cd ..
       - name: Install dependencies
         run: |
-          apm install
-          apm install linter-eslint
-          # ./node_modules/.bin/atom-package-deps .
+          pulsar --package install
+          npx atom-package-deps .
+          pulsar --package install linter-eslint
```

Comment:
I have no idea why this install does not work 😬

Reply:
I think on Windows you've still got to use `ppm` instead of `pulsar --package`, so give that a shot. If that doesn't work, I'll summon another person from Pulsar core to help troubleshoot.